bedrock/mozorg/templates/mozorg/home/home.html

```diff
@@ -38,213 +33,179 @@
 {% endblock %}
 
 {% block page_css %}
-  {{ css_bundle('home') }}
+{{ css_bundle('home') }}
 {% endblock %}
 
 {% block site_header %}
-  {% include 'mozorg/home/includes/home-icons-sprite.svg' %}
-  {% include 'includes/protocol/navigation/menu-mozilla/index.html' %}
+{% include 'includes/protocol/navigation/menu-mozilla/index.html' %}
 {% endblock %}
 
 {% block content %}
-<main role="main" class="main wrapper">
-  <section class="intro-section" id="quantum">
-    <div class="content-wrapper">
-      <div class="intro-content">
-        <strong id="fx-logo">
-        {% if l10n_has_tag('home-update-20171116') %}
-          {# L10n: the <span> below is for visual formatting only - bolding the word 'Firefox' #}
-          {{ _('The new <span>Firefox</span>') }}
-        {% else %}
-          {{ _('Firefox') }}
-        {% endif %}
-        </strong>
-
-        <h2>
-        {% if l10n_has_tag('home-update-20171116') %}
-          {{ _('Fast for good.') }}
-        {% else %}
-          {{ _('Browse faster. For good.') }}
-        {% endif %}
-        </h2>
-
-        <p>
-        {% if l10n_has_tag('home-update-20171116') %}
-          {{ _('With 2x the speed, built-in privacy protection and Mozilla behind it, the new Firefox is the better way to browse.') }}
-        {% else %}
-          {{ _('With 2x the speed, built-in privacy protection and Mozilla behind it, the new Firefox is fast for good.') }}
-        {% endif %}
-        </p>
-
-        {{ download_firefox(dom_id='download-intro', download_location='primary cta') }}
```

Comment:
It looks like there's a functional test that is relying on `dom_id='download-intro'` as a locator:

https://github.com/mozilla/bedrock/blob/master/tests/pages/home.py#L13